### ISSUE TITLE
[Feature] edit color value

### ISSUE THREAD

**Issue by rhalff** [rxlabz/panache#12]
Hi, I'm currently encountering the use case where I have copied a hex color code and want to use it as the primary theme color.

It would be nice if both the hex color and the RGB numeric values in the color picker would be editable.

What I did now is to convert the hex code to RGB manually and then try to adjust the slider to match the color.

**Comment by trietbui85** [comment]
It would be helpful if we can manually enter HEX color value.

Or if it's hard to do, we can allow to enter HEX code for each R G B value instead.

**Comment by Jeremywhiteley** [comment]
YEs!  This is needed so bad!    I want to be able to put in a color like 0xFF45C2DA.    it would add a lot of value to this app.
